### ISSUE TITLE
Global feedback

### ISSUE THREAD

**Issue by Girardvjonathan** [fuhrmanator/GIFT-grammar-PEG.js#6]
Currently global feedback (Rétroaction générale) don't get parsed (in numerical questions at least). 

## GIFT:
```
// question: 229196  name: Test Numerical
::Test Numerical::Whats two plus 4?{#
	=%100%6:0#Good job its really 6!
	####2+4 \= 6
}
```
## Result
```
Line 2, column 36: Expected {#... }, {= match1 -> Match1 ...}, {=correct choice ~incorrect choice ... }, or {T} or {F} or {True} or {False} but "{" found.
```

## Expected
```
[
   {
      "type": "Numerical",
      "title": "Test Numerical",
      "stem": "Whats two plus 4?",
      "hasEmbeddedAnswers": false,
      "choices": [
         {
            "isCorrect": true,
            "weight": 100,
            "text": {
               "type": "range",
               "number": 6,
               "range": 0
            },
            "feedback": "Good job its really 6!",
            "globalFeedBack": "2+4 = 6"
         }
      ]
   }
]
```

On my side this issues don't need to be fixed. Just wanted to notify you.

**Comment by fuhrmanator** [comment]
Thanks for the bug and the expected output. It looks like you generated this question by exporting a Moodle question to GIFT. Can you confirm that's how you did it? There are no examples on the web that I've found so far.

General feedback isn't parsed at all for now. I added the check box to #1

**Comment by Girardvjonathan** [comment]
Yes, it was exported from Moodle question to GIFT.

**Comment by fuhrmanator** [comment]
Fixed in #9
